src/api/test/functional/maintenance_test.rb

```diff
@@ -2235,7 +2235,7 @@ def test_try_to_release_without_permissions_source_permissions
     reqid = node['id']
 
     # got locked
-    get '/source/home:tom:test/_meta'
+    get '/source/home:tom:test/pack/_meta'
```

Comment:
can we use `package` in the route? `pack`means something different and it's not the abbreviation of package.

Reply:
you will need to adapt the fixtures then

Reply:
let's leave this for a rainy day...